web-app/js/portal/details/SpatialSubsetControlsPanel.js

```diff
@@ -40,43 +38,52 @@ Portal.details.SpatialSubsetControlsPanel = Ext.extend(Ext.Panel, {
             trackFiltersUsage('filtersTrackingSpatialConstraintAction', OpenLayers.i18n('trackingValueCleared'));
         }, this);
 
-        var spacer = new Ext.Spacer({
-            width: 10
-        });
-
         var pickerPanel = new Ext.Panel({
-            layout: 'hbox',
+            width: 135,
+            cls: "floatLeft",
             items: [
+                this._addLabel(OpenLayers.i18n('spatialExtentHeading')),
+                this._addVerticalSpacer(10),
                 this.polygonTypeCombo,
-                spacer,
+                this._addVerticalSpacer(5),
                 resetLink
             ]
         });
 
         this.add(pickerPanel);
-
     },
 
     _addSpatialConstraintDisplayPanel: function() {
         this.spatialConstraintDisplayPanel = new Portal.details.SpatialConstraintDisplayPanel({
             map: this.map,
-            width: this.width
+            cls: "floatLeft",
+            unstyled: true,
+            width: 160
```

Comment:
These values are specific to the layout of this component and are not referenced elsewhere or more than once each, so i dont think there is anything to gain at this stage